projects/npm-tools/packages/js-publish/src/index.js

```diff
@@ -223,7 +223,17 @@ async function runYarnPublish(pkg) {
 		args.push('--tag', 'prerelease');
 	}
 
-	run('yarn', 'publish', ...args);
+	try {
+		run('yarn', 'publish', ...args);
+	}
+	catch (error) {
+		printBanner(
+			'Failed to publish package! ❌',
+			'',
+			`You've already pushed to remote.`,
+			`Try publishing with 'yarn publish ${args.join(' ')}'`
```

Comment:
Do we get any relevant information as to why publishing failed? I'm assuming in some cases the developer might need to do something before being able to successfully re-run `yarn publish`?

Reply:
We delegate to `yarn publish` and that should emit whatever error occurred. Most likely it is due to invalid credentials or an expired OTP token. Could even be that npm is unreachable.

Rather than trying to explain the specific reason of failure (because we won't know), I thought it was best to describe what has happened (git push) and then next steps for finishing the task.

Reply:
> We delegate to yarn publish and that should emit whatever error occurred

Fair enough, then! I do like the fact that we provide clear instructions to retry 👍

Reply:
> expired OTP token

I never get the current OTP token at first try when publishing things 😂 

Adding this exception would be good instead of [a long list of instructions](https://github.com/liferay/liferay-frontend-projects/blob/6f839c15a2673c364e7afbbe9e21efae549e4993/projects/npm-tools/packages/js-publish/src/index.js#L69) :)